src/aead.rs

```diff
@@ -41,7 +41,7 @@ use crate::context::SigningTranscript;
 
 use crate::cert::AdaptorCertPublic;
 
-fn make_aead<T, AEAD>(mut t: T) -> AEAD
+pub(crate) fn make_aead<T, AEAD>(mut t: T) -> AEAD
```

Comment:
This is not an ideal solution, likely we should be sure we're doing poly1305 right and expose it from here.